README.md

```diff
@@ -62,7 +62,7 @@ cd sites/tutors-course-reader
 and copy the file `.env.example` to `.env`
 
 ```bash
-cp .\.env.example .env
```

Comment:
@bonjo7 ./ is a general notation for "look in this directory" -- yours works for sure but if someone tabs the OS tends to put ./ before it.

Reply:
Ah ok, gotcha @lgriffin , so should it not be `./.env...` and not `.\.env...`

Forward slash `/` rather than back slash `\`

Reply:
Haha actually I didn't even cop it as I done the PR to add it from my Windows machine but you are right they are flipped on Linux / Unix based systems.